.gitignore

```diff
@@ -6,3 +6,4 @@ public/
 config/pg_tileserv*.toml
 server.crt
 server.key
+hugo/.hugo_build.lock
```

Comment:
I haven't used hugo before, so just want to check with you: is it appropriate to ignore this file?